### PR TITLE
AKGF-001 update file important_note

### PR DESCRIPTION
**Pull request by Beyonder12**
Need your review.

### CODE REVIEW

File: important_note.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test
```
Comment (akigehfa) [inline important_note.txt]: what is this?

**Review by akigehfa — CHANGES_REQUESTED** [review]

File: important_note.txt (hunk 1 above)
Comment (Beyonder12) [reply important_note.txt]: This for our feature!

File: day7-181023/main.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # print(test.calculate(10, 5))
 # print(test.testVar)
-# test.Trial('Derta')
+# test.Trial('rta')
```
Comment (akigehfa) [inline day7-181023/main.py:5]: why did you change its name?

**Review by akigehfa — CHANGES_REQUESTED** [review]

File: important_note.txt (hunk 1 above)
Comment (Beyonder12) [reply important_note.txt]: Just trying!

Comment (akigehfa) [reply important_note.txt]: okay

**Review by akigehfa — APPROVED** [review]
